app/pages/sections/catalogue-solutions/order-item/flat/ondemand/ui-tests/withSavedData.ui.test.js

```diff
@@ -0,0 +1,223 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import { orderApiUrl } from '../../../../../../../config';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-id/catalogue-solutions/existing-order-id';
+
+const setCookies = ClientFunction(() => {
+  const cookieValue = JSON.stringify({
+    id: '88421113', name: 'Cool Dude', ordering: 'manage', primaryOrganisationId: 'org-id',
+  });
+
+  document.cookie = `fakeToken=${cookieValue}`;
+});
+
+const solutionNameState = ClientFunction(() => {
+  const cookieValue = 'solution-name';
+
+  document.cookie = `solutionName=${cookieValue}`;
+});
+
+const selectedRecipientIdState = ClientFunction(() => {
+  const cookieValue = 'recipient-1';
+
+  document.cookie = `selectedRecipientId=${cookieValue}`;
+});
+
+const serviceRecipientNameState = ClientFunction(() => {
+  const cookieValue = 'recipient-name';
+
+  document.cookie = `serviceRecipientName=${cookieValue}`;
+});
+
+const orderItem = {
+  serviceRecipient: {
+    odsCode: 'OX3',
+    name: 'Some service recipient 2',
+  },
+  catalogueItemType: 'Solution',
+  catalogueItemName: 'Some catalogue name',
+  catalogueItemId: '10000-001',
+  deliveryDate: '2020-04-27',
+  quantity: 3,
+  estimationPeriod: 'perMonth',
```

Comment:
this should be 'month'

Reply:
it shoudln't that doesn't match up to what is in the manifest so it doesn't autopopulate